skimage/filters/rank/generic.py

```diff
@@ -193,6 +221,9 @@ def _handle_input_3D(
         in input dtype.
     pixel_size : int, optional
         Dimension of each pixel. Default value is 1.
+    shift_x, shift_y, shift_z : int, optional
+        Offset added to the footprint center point. Shift is bounded to the
+        footprint sizes (center must be inside the given footprint).
```

Comment:
```suggestion
        footprint size (center must be inside of the given footprint).
```
Right?

Reply:
Agreed. Leaving this for you to merge. :D